Makefile.in

```diff
@@ -549,13 +551,13 @@ install: default fltools
 	rm -f $(DESTDIR)$(bindir)/spud-* $(DESTDIR)$(bindir)/diamond $(DESTDIR)$(bindir)/silenteval $(DESTDIR)$(bindir)/runut
 	[ -f manual/fluidity_manual.pdf ] && cp manual/fluidity_manual.pdf $(DESTDIR)$(docdir)/fluidity || true
 	mkdir -p $(DESTDIR)$(datadir)/fluidity
-	cp -r schemas/ $(DESTDIR)$(datadir)/fluidity/
```

Comment:
I guess it depends what the intended behaviour for this and the examples actually is. Certainly what I was seeing with BSD cp was that it was copying the contents of the source to the destination, rather than the directory itself. This then meant that the find for examples failed, because the directory it was looking in didn't exist.